client/my-sites/plans-features-main/index.jsx

```diff
@@ -166,7 +179,15 @@ class PlansFeaturesMain extends Component {
 	}
 
 	render() {
-		const { site, plans, isInSignup, sitePlans, intervalType } = this.props;
+		const {
+			site,
+			plans,
+			isInSignup,
+			sitePlans,
+			intervalType,
+			onUpgradeClick,
+			hideFreePlan = false
```

Comment:
Let's move this to default props since this is a large component.

```javascript
PlansFeaturesMain.defaultProps = {
    hideFreePlan: false
};
```

It'd be nice to also define prop types here too.